source/utils/ethUtil.ts

```diff
@@ -26,6 +26,17 @@ export const decodeTransactionData = (params: ITransactionParams) => {
 
       return decoderValue;
     }
+
+    if (!data && value) {
+      const emptyDecoderObject = {
+        method: 'Contract Interaction',
```

Comment:
The method name for transactions where there is no data but a value included should be send